tests/CommonTests.cs

```diff
@@ -35,19 +38,23 @@ public CommonTests(ITestOutputHelper helper)
 
         public static bool Docker { get; set; }
 
-
         [Fact(Timeout = Timeout)]
         public async Task CanCreateInvoice()
         {
+            const int amount = 10001;
             await WaitServersAreUp();
             foreach (var client in Tester.GetLightningClients())
             {
                 Logs.Tester.LogInformation($"{client.Name}: {nameof(CanCreateInvoice)}");
-                var createdInvoice = await client.Client.CreateInvoice(10001, "CanCreateInvoice", TimeSpan.FromMinutes(5));
+                var expectedAmount = client.Client is LndHubLightningClient 
```

Comment:
Ok, will update this as well.

Reply:
Just checked - internally I'm doing it everywhere like proposed. Here in this test I need to instantiate another value for the `expectedAmount` comparison, because the general one has an `msat` value, which would fail for LNDhub.